pkg/api/v1alpha1/cluster_webhook_test.go

```diff
@@ -903,7 +903,7 @@ func TestClusterValidateUpdateGitOpsRefOldEmptyImmutable(t *testing.T) {
 	}
 
 	g := NewWithT(t)
-	g.Expect(c.ValidateUpdate(cOld)).NotTo(Succeed())
+	g.Expect(c.ValidateUpdate(cOld)).To(Succeed())
```

Comment:
mind updating this test method's name? `TestClusterValidateUpdateIdentityProviderRefOldEmptyImmutable`